frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/HPA _Topology_&_Sidebar.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Add HPA action and topology sidebar modifications
+	As a user, I want to add a HPA to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+    And user has a workload with HPA assigned to it
+
+@regression @manual
+Scenario: Add HPA Sidebar
+   Given user is in topology
+   When user opens sidebar of workload
+   And user sees Horizontal Pod Autoscalers section under resource tab
+   And user opens action menu
+   And user can not see Edit Pod Count option 
```

Comment:
```suggestion
   And user does not see Edit Pod Count option 
```

Reply:
It will be `can not` only as the user is testing what changes happen due to HPA

Reply:
Okay.